open_flamingo/src/factory.py

```diff
@@ -87,7 +87,6 @@ def set_input_embeddings(self, new_embeddings):
     if decoder_layers_attr_name is None:
         decoder_layers_attr_name = _infer_decoder_layers_attr_name(lang_encoder)
     lang_encoder.set_decoder_layers_attr_name(decoder_layers_attr_name)
-    lang_encoder.resize_token_embeddings(len(text_tokenizer))
 
     model = Flamingo(
```

Comment:
With the new embeddings method we need to convert the old checkpoints into this format? Or maybe have the new embeds be behind a flag and say something about how this is the desired option for new models?

Reply:
Yeah, good question. My vote would be to preserve backward compatibility for now & place the new embeds behind a flag (default set to True)